getfoundation.com.website.json

```diff
@@ -0,0 +1,20 @@
+{
+  "providerId": "getfoundation.com",
+  "providerName": "Foundation Commerce Inc",
+  "serviceId": "foundation-wholesale",
```

Comment:
AssertionError: File name getfoundation.com.website.json not matching the pattern. Expected getfoundation.com.foundation-wholesale.json
E

Reply:
Name changed.